content/200-orm/500-reference/200-prisma-cli-reference.mdx

```diff
@@ -1342,6 +1342,14 @@ Other options:
    --script > script.sql
   ```
 
+## Prisma data platform
+
+### <inlinecode>platform</inlinecode>
+
+You can access the Prisma Data Platform through the Prisma CLI. To get started, ensure you have the [Prisma Client](/concepts/components/prisma-client) updated to version `5.7.0` or later. This is necessary to access the Platform through the Prisma CLI.
```

Comment:
```suggestion
The `platform` command provides access to the Prisma Data Platform through the Prisma CLI. To get started, ensure you have the [Prisma Client](/concepts/components/prisma-client) updated to version `5.7.0` or later. This is necessary to access the Platform through the Prisma CLI.
```
I noticed the other commands on this page mostly start by describing the command. Just a potential suggestion